Source/WebKit/GPUProcess/cocoa/GPUProcessCocoa.mm

```diff
@@ -95,9 +95,10 @@
 {
     RELEASE_LOG(WebRTC, "GPUProcess::ensureAVCaptureServerConnection: Entering.");
 #if HAVE(AVCAPTUREDEVICE) && HAVE(AVSAMPLEBUFFERVIDEOOUTPUT)
-    if ([PAL::getAVCaptureDeviceClass() respondsToSelector:@selector(ensureServerConnection)]) {
+    RetainPtr deviceClass = PAL::getAVCaptureDeviceClass();
```

Comment:
@rniwa I think at some point we discussed having the static analyzer understand that `Class` objects are immortal in ObjC, no?

Reply:
Indeed, would be nice to not have to do this change, also more places would need to be updated.